prow-jobs/pingcap-tidb-engine-release-6.1-latest-presubmits.yaml

```diff
@@ -0,0 +1,41 @@
+# struct ref: https://pkg.go.dev/k8s.io/test-infra/prow/config#Presubmit
+presubmits:
+  pingcap/tidb-engine-ext:
+    - name: pull-unit-test # TODO: Need to confirm if the same name can be used in different presubmit files.
+      cluster: gcp-prow-ksyun
+      decorate: true # need add this.
+      always_run: true
+      context: pull-unit-test
+      trigger: "(?m)^/test (?:.*? )?pull-unit-test(?: .*?)?$"
+      rerun_command: "/test pull-unit-test"
```

Comment:
You can omit these fields, bot can automatic set them.

Reply:
I prefer to explicitly configure it here.